payments-core/src/main/java/com/stripe/android/paymentsheet/model/StripeIntentValidator.kt

```diff
@@ -18,7 +18,7 @@ internal class StripeIntentValidator {
                 error(
```

Comment:
While you're on this, can we convert this into a plain file and static methods since the class doesn't carry any states? Or just inject it and avoid calling the constructor in `PaymentSheetViewModel` or `DefaultFlowControllerInitializer`

Reply:
Done